src/about_alan.md

```diff
@@ -3,17 +3,17 @@
 Alan is an implicitly-parallel, statically-compiled, event-based, type-inferred language with a familiar syntax and many compile-time and run-time safety guarantees.
 
 * The compiler can determine which array operations are safely parallelizable and automatically distributes them across a threadpool.
-* The runtime can determine which operations are IO operations and automatically `async/await` them, as well as automatically batch IO operations that can be executed in parallel safely.
-* The runtime has automatic memory management without a GC made possible by the language scoping semantics.
+* The Rust AVM can determine which operations are IO operations and automatically `async/await` them, as well as automatically batch IO operations that can be executed in parallel safely.
```

Comment:
The user of Alan probably doesn't care that it's written in Rust? Do you know what language the JVM is written in?

Reply:
I would care. This is the section to include details. Also Java does not transpile to Javascript as part of it's core functionality. I also put it in there to make it more clear that Javascript is not our only compilation target. Remember how around our initial launch people would get confused about whether we only target JS?

Reply:
But then they might think that we transpile to Rust?

Reply:
I think it is harder to make that logical jump since we never use the word Rust and transpile in the same sentence.

Reply:
> I think the most confusing part is "automatically `async/await` them" - implies that there's some rusty black magic to get some weird handling of Futures

That is interesting. How would you frame it? I can see why it would sound confusing, but can't think of another way to phrase that

Reply:
I think I wrote that piece a long time ago. The implied bit there was "without needing you to declare it in your code" not "because that's hard to do in Rust", which I think is what you're picking up because of the newfound emphasis on the AVM being written in Rust?

Reply:
I can just do "The AVM can determine which operations are IO operations and automatically `async/await` them without needing you to declare it in your code" ?

Reply:
Eh, I'd say something like "The AVM can determine which operations are IO operations and will `await` them accordingly"